service-admin/src/App/src/Middleware/UserValidation/UserValidationMiddleware.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
```

Comment:
Going forward with PSR-12 (https://www.php-fig.org/psr/psr-12/) you'll only separate lines in the start of a file with one linebreak. We already do this everywhere anyway so one of these should be dropped.